modules/apps/asset/asset-list-web/test/js/components/VariationsNav.test.js

```diff
@@ -31,6 +32,12 @@ const _getComponent = (props) => {
 	return <VariationsNav {...props} />;
 };
 
+jest.mock('frontend-js-web', () => ({
+	openToast: jest.fn(),
+}));
```

Comment:
Can we move this into a `beforeAll` and then in `afterAll` we can make sure to unmock this module with `jest.unmock('frontend-js-web')`.

I _think_ the mock should automatically unmock after the file is completed, but just to be sure I think it is best to be verbose with our mock and unmock for the test

Reply:
`beforeAll` produces an error relating to a promise (the same error that was the reasoning for the `Liferay.component` mock.

`afterAll` is working fine. I'll push this as a commit on top so you can see @bryceosterhaus